src/M6Web/Component/RedisMock/RedisMock.php

```diff
@@ -1079,4 +1079,41 @@ protected function deleteOnTtlExpired($key)
 
         return false;
     }
+
+    /**
+     * Mock the `quit` command
+     * @see https://redis.io/commands/quit
+     */
+    public function quit(): string
+    {
+        return 'OK'; // see the doc : always return `OK` ...
+    }
+
+    /**
+     * Mock the `monitor` command
+     * @see https://redis.io/commands/monitor
+     */
+    public function monitor()
+    {
+        return;
+    }
+
+    /**
+     * Mock the `scan` command
+     * @see https://redis.io/commands/scan
+     */
+    public function scan(): array
+    {
+        return [];
```

Comment:
If you return a empty array, how can you use the mock ? You should scan the dataValues and look into deleteOnTtlExpired.